Data/IDayOffRequestRepository.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using stunning_robot_HR.Models;
+
+namespace stunning_robot_HR.Data
+{
+    public interface IDayOffRequestRepository
```

Comment:
Nice!!!!! Now you're getting to the last letter of SOLID, dependency inversion principle. You'll have your controllers depending on the interface IDayOffRequestRepository rather than the class that implements it. Why is that important, a little concept known as polymorphism. Ultimately it means that you could swap any implementation of IDayOffRequestRepository with another, and none of the code in your controller changes at all. It's a really powerful concepts. It effectively means your code is loosely coupled. Rather than your controllers depending on a concrete class they effectively way hey, as long as you have the methods of IDayOffRequestRepository than you can play here. Nice man...